experiments/bo_options.py

```diff
@@ -5,28 +5,58 @@ def lora_target_map(model: str):
                 'qkv_proj': ['k_proj', 'q_proj', 'v_proj'],
                 'attn_head': ['k_proj', 'q_proj', 'v_proj', 'dense'],
                 'attn_head_and_mlp': ['k_proj', 'q_proj', 'v_proj', 'dense', 'fc1', 'fc2'],
```

Comment:
Add `attn_shortcut_Q` here? Because that would make comparing to paper experiments easiest (in the paper we didn't finetune the lm head)

Reply:
Did the paper use 'k_proj', 'q_proj', 'v_proj', 'dense', 'fc1', 'fc2'? If so, I'll preserve that as an option.

Reply:
That's right, sounds good.

Reply:
I now use both shortcut_Qs for serial and the single shortcut_Q for parallel architectures whenever they are used in a lora target option. I think this addresses most of the comments now.